tests/test_rooms/test_daily_room_events.py

```diff
@@ -0,0 +1,13 @@
+from tests.base import BaseTestCase, CommonTestCases
+from fixtures.room.daily_room_events_fixture import (
+    daily_room_events_query, daily_room_events_response
+)
+
+
+class TestDailyEvents(BaseTestCase):
+    def test_analytics_for_daily_events(self):
+        CommonTestCases.admin_token_assert_equal(
+            self,
+            daily_room_events_query,
+            daily_room_events_response
+        )
```

Comment:
You could add a test for wrong date format @caveinn

Reply:
I have implemented that @bevkololi